### DISCUSSION TITLE
Where is the SDK config file format documented?

### DISCUSSION THREAD

**Discussion by rpgdev** [open-telemetry/opentelemetry-java-instrumentation#11813] [Q&A] [answered]
Going over the javaagent example I can see that they use the following config file for the SDK: https://github.com/open-telemetry/opentelemetry-java-examples/blob/main/javaagent/sdk-config.yaml but I can't find any documentation regarding the format of that file, how did you know to use `tracer_provider` etc? I know they mention it here https://opentelemetry.io/docs/specs/otel/configuration/file-configuration/#configuration-file but it doesn't go that deep into it.

**Comment by jkwatson** [comment]
since this is configuration of the agent, I'm going to move this over to the instrumentation repository.

**Comment by trask** [comment, marked as answer]
it is defined here: https://github.com/open-telemetry/opentelemetry-configuration

but note that this is very early experimental yaml configuration
